prisma/seed.ts

```diff
@@ -136,6 +138,11 @@ async function main() {
           Math.floor(Math.random() ** 1.5 * 90 * 24 * 60 * 60 * 1000),
       );
 
+      const envTag = envTags[Math.floor(Math.random() * envTags.length)];
+      const colorTag = colorTags[Math.floor(Math.random() * colorTags.length)];
+
+      const tags = [envTag, colorTag].filter((tag) => tag !== "");
```

Comment:
nit: i think it would be nicer to have null in the envTags or colorTags and then filter by them. This way we dont have the risk of having empty strings in the array, which we might not see.

Reply:
Not sure if I misunderstood you but just to clarify in L144 we are filtering out any empty string. When envTag and colorTag are both [""], after filtering both the const tags will be [], which is the default of our prisma schema.